_includes/component-pattern-overview.html

```diff
@@ -1,9 +1,9 @@
 <div class="wrapper wrapper-fluid">
     <header>
       <h1 class="title">{{page.title}}</h1>
-      <h4>{{page.description}}</h4>
+      <label for="search" class="h4">{{page.description}}</label>
```

Comment:
Use <div> instead. The page description is not part of the components' search bar
```suggestion
      <div class="h4">{{page.description}}</div>
```